### PR TITLE
Added metrics reader access to prometheus

### PR DESCRIPTION
**Pull request by bamachrn**
Signed-off-by: Bama Charan Kundu <bamachrn@gmail.com>

### CODE REVIEW

File: components/monitoring/prometheus/base/configure-prometheus.yaml [hunk 1]
```diff
@@ -284,4 +284,45 @@ subjects:
...
+  name: host-operator-metrics-reader
+subjects:
+- kind: ServiceAccount
+  name: prometheus-k8s
+  namespace: appstudio-workload-monitoring
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  creationTimestamp: null
```
Comment (xcoulon) [inline components/monitoring/prometheus/base/configure-prometheus.yaml:305]: same here

File: components/monitoring/prometheus/base/configure-prometheus.yaml [hunk 2]
```diff
@@ -284,4 +284,45 @@ subjects:
...
+  name: member-operator-metrics-reader
+subjects:
+- kind: ServiceAccount
+  name: prometheus-k8s
+  namespace: appstudio-workload-monitoring
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  creationTimestamp: null
```
Comment (xcoulon) [inline components/monitoring/prometheus/base/configure-prometheus.yaml:319]: same here

**Review by xcoulon — CHANGES_REQUESTED** [review]

**Review by xcoulon — APPROVED** [review]

**Comment by amfred** [comment]
This'll need a README update if it works.

**Pull request merged**